pyproject.toml

```diff
@@ -56,4 +56,5 @@ F3D_WINDOWS_BUILD_SHELL_THUMBNAILS_EXTENSION = "OFF"
 F3D_ENABLE_SOVERSION = "OFF"
 
 [tool.pytest.ini_options]
-python_files = "python/testing/*.py"
+addopts = "-rA"
+filterwarnings = "ignore::DeprecationWarning"
```

Comment:
what does these two lines do ?

Reply:
A `DeprecationWarning` is raised when using Python deprecated functions. We are testing them so it's very noisy. This line hides the warnings and makes the pytest output clearer.